cvmfs/cvmfs_server

```diff
@@ -54,21 +54,44 @@ publish_after_hook() { :; }
 
 [ -f /etc/cvmfs/cvmfs_server_hooks.sh ] && . /etc/cvmfs/cvmfs_server_hooks.sh
 
+find_sbin() {
+  local bin_name="$1"
+  local bin_path=""
+  for d in /sbin /usr/sbin /usr/local/sbin /bin /usr/bin /usr/local/bin; do
```

Comment:
mmh, for the moment we would look for the following tools:
* `a2enmod`
* `apache2`
* `apache2ctl`
* `apachectl`
* `getcap`
* `getenforce`
* `httpd2`
* `lsof`
* `modprobe`
* `pidof`
* `runuser`
* `sestatus`
* `set-up`

Most of it is clearly system stuff which probably wouldn't end up in `/usr/local`. Only for apache I am not so sure...